README.md

```diff
@@ -48,7 +48,6 @@ If the basic registers change you can restart your register so that it sees the
 - Install and run postgres db
 - Checkout project
 - Run the `./go` script to set everything up
-- `$ git submodule init`
 - `$ git submodule update`
```

Comment:
Let's remove this line and instead edit the `go` script to replace:

`git submodule init`

with 

`git submodule update --init`

Which will 'update' submodules to the expected version and initialize any that haven't yet been initialized.